swarm.js

```diff
@@ -22,7 +22,10 @@ function swarm (m, onjoin, opts) {
   const announce = market.isSeller(m)
   const lookup = !announce
 
-  m.ready(() => swarm.join(m.discoveryKey, { announce, lookup }, onjoin))
+  m.ready(() => {
+    if (announce && !m.uniqueFeed) swarm.join(m.feed.discoveryKey, { announce, lookup })
```

Comment:
is an early return required?

Reply:
hmmm... on a uniquefeed `m.uniqueFeed === undefined` for me.

here is my setup:

```js
const dopts = { masterKey: keys.masterKey }
const market = dazaar(path.join(__dirname, 'dbs', 'dazaar', name), dopts)

let feed
market.ready(() => {
  feed = hypercore(
    path.join(__dirname, 'dbs', 'core', name),
    keyPair.publicKey,
    { stats, secretKey: keyPair.secretKey }
  )

  feed.ready(next)
})


function next () {
  seller = market.sell(feed, {
    validate (remoteKey, done) {
      payment.validate(remoteKey, function (err, info) {
        done(err, info)
      })
    },
    uniqueFeed: false
  })
}
```

Reply:
Thanks, will investigate

Reply:
I think this is solved now with a reinstall (was a hyperbee bug)